budget/config.py

```diff
@@ -0,0 +1,166 @@
+import os
+from pathlib import Path
+
+from cerberus import Validator
+import yaml
+
+class Config:
+
+  _budget_rules_schema = {
+    'teams': {
```

Comment:
@zaro0508 good idea; any idea what those values are?

Reply:
the min and max values?  i don't know but i'm guessing @brucehoff would know

Reply:
Team IDs come from an index which starts at 1 and continues to grow with no upper limit.  Even though you typically see team IDs which are 7 digit numbers it would be incorrect to set that as a constraint.

Reply:
I think actually calling the API would be taking the validation a bit too far. Does anyone think that's necessary? How about I check for a minlength of 1?